gravitee-apim-console-webui/src/entities/management-api-v2/environment-flows/EnvironmentFlows.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// TODO: complete the EnvironmentFlows interface when the OpenAPI is available
+export interface EnvironmentFlows {
+  name?: string;
+  description?: string;
+  phase?: string;
+  updatedAt?: Date;
+  deployedAt?: Date;
```

Comment:
the "configuration" will come in another PR I guess, but maybe you can already prepare the `prerequisite` field ?

Reply:
A yes Hum for me it's not yet clear this field I think it's a separate US topic that can be added BackEnd and Front at the same time. WDYT ? 
I've included these because they appear in the table as examples.